app/js/HealthChecker.js

```diff
@@ -60,8 +63,14 @@ module.exports = {
           }
         })
       },
-      (cb) => db.docs.remove({ _id: doc_id, project_id }, cb),
-      (cb) => db.docOps.remove({ doc_id }, cb)
+      (cb) =>
+        docsCollectionPromise.then((docs) =>
```

Comment:
Interesting. Keeping the promise around and calling `then` whenever we want the return value. Nice pattern.